docs/elements/examples/shadcn-ui.mdx

```diff
@@ -0,0 +1,508 @@
+---
+description: Learn how to compose Clerk Elements with shadcn/ui to build custom sign in and sign up flows.
+---
+
+# shadcn/ui
+
+The following examples demonstrate how to compose Clerk Elements with shadcn/ui to build custom sign-in and sign-up flows.
+## Prerequisites
```

Comment:
We don't use this term anywhere else in the docs - I would prefer "Before you start" because that's we also use in the `<TutorialHero />` component

@S3Prototype what do you think ?

Reply:
That makes sense, yeah.

Reply:
```suggestion
## Before you start
```